packages/bumbag/src/SelectMenu/SelectMenu.tsx

```diff
@@ -801,7 +802,7 @@ function SelectMenuSearchInput(props: any) {
         overrides={overrides}
         placeholder="Type to search..."
         value={value}
-        autoFocus={autoFocus}
```

Comment:
We need to keep this, don't we?

Reply:
I was thinking autoFocus is a HTML attribute and not our onFoucs prop?